packages/lesswrong/lib/collections/lwevents/views.ts

```diff
@@ -63,7 +63,13 @@ LWEvents.addView("gatherTownUsers", (terms: LWEventsViewTerms) => {
 })
 
 // Index used in manual user-by-IP queries, and in some moderator UI
-ensureIndex(LWEvents, {name:1, "properties.ip":1, createdAt:1, userId:1})
+ensureCustomPgIndex(`
+  CREATE INDEX CONCURRENTLY "manual_idx__LWEvents_properties_ip"
```

Comment:
`CREATE INDEX CONCURRENTLY IF NOT EXISTS`?

Reply:
Fixed in followup commit (along with a migration, which turns out to be necessary now)